CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -320,3 +282,104 @@ Some background and text colors use the `GiniColor` type with which you can set
+- Tip 2 image &#8594; <span style="color:#009EDF">*captureSuggestion2*</span> image asset
+- Tip 3 image &#8594; <span style="color:#009EDF">*captureSuggestion3*</span> image asset
+- Tip 4 image &#8594; <span style="color:#009EDF">*captureSuggestion4*</span> image asset
+- Tip 5 image &#8594; <span style="color:#009EDF">*captureSuggestion5*</span> image asset
+
+##### 3. Background
+
+- Background color &#8594; `GiniConfiguration.noResultsScreenBackgroundColor` using `GiniColor` with dark mode and light mode colors
+
+##### 4. Back to camera button
+- Background color &#8594; `GiniConfiguration.noResultsBottomButtonColor`
+- Text color &#8594; `GiniConfiguration.noResultsBottomButtonTextColor`
+- Corner radius &#8594; `GiniConfiguration.noResultsBottomButtonCornerRadius`
+- Back button
+  - With image and title
+      - Image &#8594; <span style="color:#009EDF">*cameraIcon*</span> image asset
+      - Title &#8594; <span style="color:#009EDF">*ginicapture.noresults.gotocamera*</span> localized string
+
+
+## Supported formats screen
+
+<br>
+<center><img src="img/Customization guide/Supported formats.jpg" height="500"/></center>
+</br>
+
+##### Navigation bar
+- Back button
+  - With image and title
+	  - Image &#8594; <span style="color:#009EDF">*arrowBack*</span> image asset
+	  - Title &#8594; <span style="color:#009EDF">*ginicapture.navigationbar.help.backToMenu*</span> localized string
+  - With title only
+	  - Title &#8594; `GiniConfiguration.navigationBarHelpScreenTitleBackToMenuButton`
+
+##### 1. Supported format cells
+- Supported fortmats icon &#8594; <span style="color:#009EDF">*supportedFormatsIcon*</span> image asset
+- Supported formats icon color &#8594; `GiniConfiguration.supportedFormatsIconColor`
+- Non supported fortmats icon &#8594; <span style="color:#009EDF">*nonSupportedFormatsIcon*</span> image asset
+- Non supported formats icon color &#8594; `GiniConfiguration.nonSupportedFormatsIconColor`
+
+##### 2. Table View Cells
+- Background color &#8594; `GiniConfiguration.supportedFormatsScreenCellsBackgroundColor` using `GiniColor` with dark mode and light mode colors
+
+##### 3. Background
+- Background color &#8594; `GiniConfiguration.supportedFormatsScreenBackgroundColor` using `GiniColor` with dark mode and light mode colors
+
+## Open with tutorial screen
+
+<br>
+<center><img src="img/Customization guide/Open with tutorial.jpg" height="500"/></center>
+</br>
+
+##### Navigation bar
+
+- Back button
+  - With image and title
+	  - Image &#8594; <span style="color:#009EDF">*arrowBack*</span> image asset
+	  - Title &#8594; <span style="color:#009EDF">*ginicapture.navigationbar.help.backToMenu*</span> localized string
+  - With title only
+	  - Title &#8594; `GiniConfiguration.navigationBarHelpScreenTitleBackToMenuButton`
+
+##### 1. Header
+
+- Text &#8594; <span style="color:#009EDF">*ginicapture.help.openWithTutorial.collectionHeader*</span> localized string
+
+##### 2. Open with steps
+
```

Comment:
[Suggestion] @ValentinaIancu-Gini could you please align the formatting in the sections for help screens? 🙏

Reply:
done